src/components/MessageCard/MessageCard.jsx

```diff
@@ -86,16 +99,45 @@ export class MessageCard extends React.PureComponent {
   renderImage() {
     const { image } = this.props
 
-    return image ? (
+    if (!image) {
+      return null
+    }
+
+    const { height, width } = this.calculateSize(image)
+
+    return (
       <ImageContainerUI>
         <ImageUI
           src={image.url}
           alt={image.altText || 'Message image'}
-          width={image.width || '100%'}
-          height={image.height || 'auto'}
+          width={width || '100%'}
+          height={height || 'auto'}
```

Comment:
Should we _not_ pass the default values `100%` and `auto` here, since we're considering the null scenario in the styled component? Otherwise, I think we could end up with a value like `100%px`.

Reply:
yeah - same as above :facepalm: